tests/integration/ceph_mgr_test.go

```diff
@@ -87,46 +88,46 @@ type service struct {
 	Status      serviceStatus
 }
 
-func (suite *CephMgrSuite) SetupSuite() {
-	suite.namespace = "mgr-ns"
-
-	mgrTestCluster := TestCluster{
-		clusterName:             suite.namespace,
-		namespace:               suite.namespace,
-		storeType:               "bluestore",
-		storageClassName:        "",
-		useHelm:                 false,
-		usePVC:                  false,
-		mons:                    1,
-		rbdMirrorWorkers:        0,
-		rookCephCleanup:         true,
-		skipOSDCreation:         true,
-		minimalMatrixK8sVersion: cephMasterSuiteMinimalTestVersion,
-		rookVersion:             installer.VersionMaster,
-		cephVersion:             installer.MasterVersion,
+func (s *CephMgrSuite) SetupSuite() {
+	s.namespace = "mgr-ns"
+
+	s.settings = &installer.TestCephSettings{
+		ClusterName:       s.namespace,
+		OperatorNamespace: installer.SystemNamespace(s.namespace),
+		Namespace:         s.namespace,
+		StorageClassName:  "",
+		UseHelm:           false,
+		UsePVC:            false,
+		Mons:              1,
+		RBDMirrorWorkers:  0,
```

Comment:
I feel like we can remove this option, it does not seem to be used anywhere.

Reply:
Ok, I see there is a specific test that enables it in the smoke suite, I'll remove this setting.